de.js

```diff
@@ -1,19 +1,19 @@
 {
 	"%s ago": "vor %s",
 	"%d unread message": {
-		"one": "%d unread message",
-		"other": "%d unread messages"
+		"one": "%d ungelesene Nachricht",
+		"other": "%d ungelesene Nachrichten"
 	},
 	"Unread messages in %%s, %%s and one other": {
 		"one": "Unread messages in %%s, %%s and one other",
 		"other": "Unread messages in %%s, %%s and %d others"
 	},
-	"Unread messages in %s": "Unread messages in %s",
-	"Unread messages in %s and %s": "Unread messages in %s and %s",
-	"Unread messages from %s": "Unread messages from %s",
-	"Unread messages from %s and %s": "Unread messages from %s and %s",
+	"Unread messages in %s": "Ungelesene Nachrichten in %s",
+	"Unread messages in %s and %s": "Ungelesene Nachrichten in %s and %s",
+	"Unread messages from %s": "Ungelesene Nachrichten von %s",
+	"Unread messages from %s and %s": "Ungelesene Nachrichten von %s und %s",
 	"Unread messages from %%s, %%s and one other": {
-		"one": "Unread messages from %%s, %%s and one other",
-		"other": "Unread messages from %%s, %%s and %d others"
+		"one": "Ungelesene Nachricht von %%s, %%s und einem anderen",
```

Comment:
I wonder if there’s a gender-neutral phrasing? (The only trick I can come up with is „und 1 anderen“, which looks bad.)